modules/text/src/text_detector_swt.cpp

```diff
@@ -0,0 +1,817 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "precomp.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/imgproc.hpp"
+#include <cmath>
+#include <math.h>
+#include <utility>
+#include <iostream>
+#include <algorithm>
+#include <unordered_map>
+#include <vector>
```

Comment:
Please filter out headers which are already in precomp.hpp or not used (like `iostream`)